src/models/TestResultsSchemaHGVSubmitted.js

```diff
@@ -0,0 +1,75 @@
+const Joi = require('joi')
+
+const defectsSchema = Joi.object().keys({
```

Comment:
These Schemas are Extremely similar, to the point where I'm not sure that some of the differences are supposed to be differences. Can you run some Diffs over these to make sure they are correct, and consider extracting common pieces for reuse, rather than re-creating the wheel every time?